testapi.pm

```diff
@@ -2313,4 +2315,18 @@ sub compat_args {
     return %ret;
 }
 
+=head2 show_curl_progress_meter
+
+Helper function to alter the curl command to show progress meter.
+Progress meter is shown only when the server output is redirected.
+This works only when uploading where the output is not lately used.
+
+    show_curl_progress_meter( $cmd )
+
+A typical call would look like:
+
+    $cmd .= show_curl_progress_meter($cmd);
+=cut
+sub show_curl_progress_meter { "-o /dev/$serialdev " if get_var('UPLOAD_METER') }
```

Comment:
> @okurz sorry that it took me a while… I now finally have passing test for `show_curl_progress_meter()` but:
> 
>     1. Neither `upload_assets()` neither `upload_logs()` are covered in `t/03-testapi.t` and _codecov_ is still failing.

Well, would be good to have that covered as well but I am not demanding that from you :)

>     2. When `UPLOAD_METER` is not set it returns `'0'` - isn't that problem?

well, I guess this is something you could cover with tests for `upload_assets/upload_logs` ;)

Better make the function always return something well defined, e.g.

```suggestion
sub show_curl_progress_meter { get_var('UPLOAD_METER') ? "-o /dev/$serialdev " : '' }
```